contributors.js

```diff
@@ -13,5 +13,6 @@ const contributors = [
   { username: 'yannbertrand', message: 'Hello, world!' },
   { username: 'adrilabelle', message: 'First Open Source baby!' },
   { username: 'gudsfile', message: 'Ça écoute Zenikast oéoé' },
-	{ username: 'SahilShaw2004', message: 'First pr hopefully for hacktoberfest' }
+	{ username: 'SahilShaw2004', message: 'First pr hopefully for hacktoberfest' },
+	{ username: 'zeroual', message: 'Hellow hacktoberfest from Morocco!' }
```

Comment:
Hello, if you use your username zeroual, your duck file have to zeroual.png :)

Reply:
hello @jeanphi-baconnais  , i fixed that. thnaks